docs/translations/api-docs/hidden/hidden.json

```diff
@@ -8,37 +8,17 @@
     "initialWidth": {
       "description": "You can use this prop when choosing the <code>js</code> implementation with server-side rendering.<br>As <code>window.innerWidth</code> is unavailable on the server, we default to rendering an empty component during the first mount. You might want to use a heuristic to approximate the screen width of the client browser screen width.<br>For instance, you could be using the user-agent or the client-hints. <a href=\"https://caniuse.com/#search=client%20hint\">https://caniuse.com/#search=client%20hint</a>"
     },
-    "lgDown": {
-      "description": "If <code>true</code>, component is hidden on screens below (but not including) this size."
```

Comment:
IMO the old description made more sense. We could update the JSDoc in .d.ts to match the one in .js so the generated API docs don't change.

Reply:
I updated the JSDoc of the type :+1: